tests/system_tests/external_interaction/test_ispyb_dev_connection.py

```diff
@@ -662,10 +662,10 @@ def test_ispyb_deposition_in_gridscan(
     assert position_id is None
     DC_EXPECTED_VALUES.update(
         {
-            "axisstart": 90.0,
-            "axisend": 90.0,
+            "axisstart": -90.0,
+            "axisend": -90.0,
             "datacollectionnumber": 2,
-            "omegastart": 90.0,
+            "omegastart": -90.0,
```

Comment:
Must: Sorry, I know we discussed this but upon reflection I think we should make sure that we get the sign right in this PR (maybe just invert it in the callback?) then we can remove in the follow up issue. Otherwise if we release this before getting round to the next issue we break analysis